src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -1207,7 +1213,7 @@ class LibraryServiceApi
      * try {
      *     $libraryServiceApi = new LibraryServiceApi();
      *     $formattedName = LibraryServiceApi::formatBookName("[SHELF]", "[BOOK]");
-     *     $indexName = "";
+     *     $indexName = "default index";
```

Comment:
How did this happen? Was it previously supposed to be like this, and you fixed a bug in the init code handling?